taxi/forms.py

```diff
@@ -0,0 +1,38 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver
+
+
+def validate_license_number(license_number):
+    if len(license_number) == 8 \
+            and license_number[:3].isalpha() \
+            and license_number[:3] == license_number[:3].upper() \
+            and license_number[3:].isdigit():

```

Comment:
Don't use `/`, use `( )` for the conditionals. Check out the conditional style here https://peps.python.org/pep-0008/#indentation.